cobra_vendor/CMakeLists.txt

```diff
@@ -9,30 +9,35 @@ if(DEFINED CMAKE_BUILD_TYPE)
   list(APPEND extra_cmake_args "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}")
 endif()
 
+# The Cobra source code version/revision numbers
+set(VER "3.9")
+set(REV "02518bf8f45e398a22ac3923716d8e7c78bc83c3")
+
 include(ExternalProject)
-ExternalProject_Add(cobra-3.6
-  PREFIX cobra-3.6
+ExternalProject_Add(cobra-${VER}
+  PREFIX cobra-${VER}
   GIT_REPOSITORY https://github.com/nimble-code/cobra
-  GIT_TAG ab06e676f9123ec07011a09c38daffaa4fbfdc60
+  GIT_TAG ${REV}
   INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/cobra_install"
   BUILD_IN_SOURCE TRUE
   CONFIGURE_COMMAND ""
   BUILD_COMMAND
   ${CMAKE_COMMAND} -E chdir <SOURCE_DIR>/src make linux && ${CMAKE_COMMAND} -E chdir <SOURCE_DIR>/src_app make all
   INSTALL_COMMAND
-  ${CMAKE_COMMAND} -E chdir <SOURCE_DIR>/src make install_linux MAN=/dev/null && ${CMAKE_COMMAND} -E chdir <SOURCE_DIR>/src_app make install_linux
+  ${CMAKE_COMMAND} -E chdir <SOURCE_DIR>/src make install_linux && ${CMAKE_COMMAND} -E chdir <SOURCE_DIR>/src_app make install_linux
```

Comment:
Why was the manpage installation variable changed?

The Space ROS CI build is broken since it can't install the manpages to the default `/usr/share` nor should this vendor package be installing things to anywhere outside the specified prefix.

Reply:
I've opened https://github.com/ament/ament_cobra/pull/7 tackling the regression. Clearly that line worked for me in some situations as it stayed in for some time, but it is no longer working so I've made an adjustment.

Reply:
It was failing for me, not able to install the man pages to /dev/null (a file); it needed a directory.